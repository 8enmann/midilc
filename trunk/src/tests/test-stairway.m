main(){
  set_instrument("Voice Oohs");
  set_tempo(80);
  play(stairway1() + stairway2() + stairway1() + stairway3());
}

stairway1(){
  Sequence s;
  s = new_sequence();
  s = s + new_chord(C5e, A3e) + C4e + E4e + A4e;
  s = s + new_chord(B4e, G#e) + E4e + C4e + B4e;
  s = s + new_chord(C5e, Ge) + E4e + C4e + C5e;
  s = s + new_chord(F#4e, F#3e) + D4e + A3e + F#4e;
  s = s + new_chord(E4e, F3e) + C4e + A3e + C4q;
  s = s + E4e + C4e + A3e;
  s = s + new_chord(A2e, D3e, G3e, B3e) + new_chord(A2e, E3e, A3e, C4e) + new_chord(A2q, E3e, A3e, C4e);
  return s;
}

stairway2(){
  Sequence f;
  f = new_sequence() + Re + A2e + F3e + E3e;
  return f;
}

stairway3(){
  Sequence f;
  f = new_sequence() + Rq + A2e + B2e;
  return f;
}
