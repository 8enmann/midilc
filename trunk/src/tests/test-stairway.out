Instrument,Voice Oohs
Tempo,80
0,2,72
0,2,57
2,2,60
4,2,64
6,2,69
8,2,71
8,2,68
10,2,64
12,2,60
14,2,71
16,2,72
16,2,67
18,2,64
20,2,60
22,2,72
24,2,66
24,2,54
26,2,62
28,2,57
30,2,66
32,2,64
32,2,53
34,2,60
36,2,57
38,4,60
42,2,64
44,2,60
46,2,57
48,2,45
48,2,50
48,2,55
48,2,59
50,2,45
50,2,52
50,2,57
50,2,60
52,4,45
52,4,52
52,4,57
52,4,60

58,2,45
60,2,53
62,2,52
64,2,72
64,2,57
66,2,60
68,2,64
70,2,69
72,2,71
72,2,68
74,2,64
76,2,60
78,2,71
80,2,72
80,2,67
82,2,64
84,2,60
86,2,72
88,2,66
88,2,54
90,2,62
92,2,57
94,2,66
96,2,64
96,2,53
98,2,60
100,2,57
102,4,60
106,2,64
108,2,60
110,2,57
112,2,45
112,2,50
112,2,55
112,2,59
114,2,45
114,2,52
114,2,57
114,2,60
116,4,45
116,4,52
116,4,57
116,4,60

124,2,45
126,2,47

